docs/content/en/docs/intro/_index.md

```diff
@@ -1,173 +1,124 @@
-we discussed the differences and similarities
-between Keptn v1 and Keptn
-to help you decide which best fits your needs.
-View the recording:
-[Compare Keptn v1 and the Keptn Lifecycle Toolkit](https://www.youtube.com/watch?v=-cKyUKFjtwE&t=170s)
-
-## Overviews of Keptn
-
-A number of presentations are available to help you understand Keptn.
-Note that many of these refer to the "Keptn Lifecycle Controller"
-or "Keptn Lifecycle Toolkit", which was the development code name for Keptn.
-
-* [Orchestrating and Observing GitOps Deployments with Keptn](https://www.youtube.com/watch?v=-cKyUKFjtwE&t=11s)
-  discusses the evolution of Keptn and the concepts that drive it,
-  then gives a simple demonstration of a Keptn implementation.
-
-* [Introducing Keptn Lifecycle Toolkit](https://youtu.be/449HAFYkUlY)
-  gives an overview of what Keptn does and how to implement it.
-
-* [Keptn Lifecycle Toolkit Demo Tutorial on k3s, with ArgoCD for GitOps, OTel, Prometheus and Grafana](https://www.youtube.com/watch?v=6J_RzpmXoCc)
-  is a short video demonstration of how Keptn works.
-  You can download the exercise and run it for yourself;
-  notes below the video give a link to the github repo.
-  The README file in that repo gives instructions for installing the software
-  either automatically or manually.
-
-* [What is the Keptn Lifecycle Toolkit?](https://isitobservable.io/observability/kubernetes/what-is-the-keptn-lifecycle-toolkit)
-  blog discusses Keptn as part of the "Is It Observable?" series.
-  This links to:
-
-* [What is Keptn Lifecycle Toolkit?](https://www.youtube.com/watch?v=Uvg4uG8AbFg)
-  is a video that steps through the process of integrating Keptn
-  with your existing cloud native cluster.
-
-* [Keptn Lifecycle Toolkit: Installation and KeptnTask Creation in Minutes](https://www.youtube.com/watch?v=Hh01bBwZ_qM)
-  demonstrates how to install Keptn and create your first KeptnTask in less than ten minutes.
-  
-* You can explore the [GitHub repository](https://github.com/isItObservable/keptn-lifecycle-Toolkit)
-  for more information.
+## Release lifecycle management
+
+The Release lifecycle management tools run in conjunction
+with the standard Kubernetes deployment tools
+to make deployments more robust.
+
+These tools run checks and tasks before or after deployment initiation.
+
+* Pre-deployment tasks such as checking for dependent services,
+  image scanning, and setting the cluster to be ready for the deployment
+
+* Pre-deployment evaluations such as checking the layout of the cluster
+
+* Post-deployment tasks such as triggering tests,
+  triggering a deployment to another cluster,
+  or sending notifications that the deployment succeeded or failed
+
+* Post-deployment evaluations to evaluate the deployment,
+  evaluate test results,
+  or confirm software health against SLOs
+  like performance and user experience
+
+All `KeptnTask` resources that are defined by `KeptnTaskDefinition` resources
```

Comment:
I think this and the next paragraph don't really fit in here nicely. This is the first time you talk about keptn specific resources and the execution order and detailed features are not needed in this intro I think.
I suggest to just remove those paragraphs, since this info also exists on other pages AFAIK.

Reply:
I'm not sure about this -- part of a bigger question about what exactly this page should be -- should it be just high-level stuff with nowhere to go or an entry point to the rest of the information.  We're ready to publish so let's address that question later.